qualtran/bloqs/qft/qft_text_book.py

```diff
@@ -56,3 +81,25 @@ def decompose_from_registers(
         if self.with_reverse:
             for i in range(self.bitsize // 2):
                 yield cirq.SWAP(q[i], q[-i - 1])
+
+    def build_call_graph(self, ssa: SympySymbolAllocator) -> Set['BloqCountT']:
+        ret = {
+            (Hadamard(), self.bitsize),
+            (
+                PhaseGradientUnitary(self.bitsize - 1, exponent=0.5, is_controlled=True),
+                self.bitsize - 1,
```

Comment:
This would be incorrect when the bloq is not parameterized. If `self.bitsize` is parameterized, we can return a single bloq `self.bitsize-1` times, but if thats not the case we should return `self.bitsize-1` different bloqs, each with count 1. 

We've followed this approach in other bloqs, like `LPRSInterimPrep.build_call_graph`. For the non-symbolic case we should aim the `build_call_graph` to return accurate counts and we can make some approximations for the symbolic case where its harder to express such sequence of bloqs.

Reply:
this is very helpful, the non-symbolic case makes sense I will go ahead and update that but I'm not sure what kind of approximation you mean for the symbolic case. Would a worst case count be sufficient?

Reply:
I think "approximation" here just means you don't know the exact parameters/attributes for the callee bloqs